make-release.sh

```diff
@@ -184,63 +186,79 @@ replaceFieldSed()
   replaceFieldSed "${YAMLFILE}" 'prod-ver-major' "\"${MAJOR}\""
-  replaceFieldSed "${YAMLFILE}" 'prod-ver' "\"${MAJOR}.${MINOR}\""
-  replaceFieldSed "${YAMLFILE}" 'prod-ver-patch' "\"${MAJOR}.${MINOR}.${PATCH}\""
-  replaceFieldSed "${YAMLFILE}" 'prod-prev-ver' "\"${MAJOR}.$((MINOR - 1))\""
+  replaceFieldSed "${YAMLFILE}" 'prod-ver' "\"${MAJOR}.$((MINOR + 1))\""
+  replaceFieldSed "${YAMLFILE}" 'prod-ver-patch' "\"${MAJOR}.$((MINOR + 1)).0\""
+  replaceFieldSed "${YAMLFILE}" 'prod-prev-ver' "\"${MAJOR}.${MINOR}\""
   # Update the version, defined in the antora-playbook-for-publication.yml file, in following keys:
   #    branches: 7.32.x
   #    edit_url: "https://github.com/eclipse/che-docs/edit/7.35.x/{path}"
+  fi  
+  echo "[INFO] Generating single-sourced docs on branch: ${MAIN_BRANCH}."
+  ./tools/checluster_docs_gen.sh
+  ./tools/environment_docs_gen.sh
+  echo "[INFO] Finished handling version update on branch: ${MAIN_BRANCH}."
+  gitPullRequest ${MAIN_BRANCH} ${HEAD_BRANCH}
+}
+
+publicationsBuilderUpdate() {
+  git checkout ${PUBLICATION_BRANCH}
   YAMLFILE=antora-playbook-for-publication.yml
   if [ -f "${YAMLFILE}" ]
   then
-    replaceFieldSed "${YAMLFILE}" 'branches' "\"${MAJOR}.${MINOR}.x\""
-    replaceFieldSed "${YAMLFILE}" 'edit_url' "\"https://github.com/eclipse/che-docs/edit/${MAJOR}.${MINOR}.x/{path}\""
+    sed "/   - branches:/a ${MAJOR}.${MINOR}.x\}" ${YAMLFILE}
   else
-    echo "[WARNING] Cannot find file: ${YAMLFILE} on branch: ${TARGET_BRANCH}. Skipping."
-  fi  
-  echo "[INFO] Generating single-sourced docs on branch: ${TARGET_BRANCH}."
-  ./tools/checluster_docs_gen.sh
-  ./tools/environment_docs_gen.sh
-  echo "[INFO] Finished handling version update on branch: ${TARGET_BRANCH}.."
+    echo "[WARNING] Cannot find file: ${YAMLFILE} on branch: ${MAIN_BRANCH}. Skipping."
+  echo "[INFO] Finished handling version update on branch: ${PUBLICATION_BRANCH}"
+  gitPush ${PUBLICATION_BRANCH}
 }
 
+patchVersionUpdate() {
+  checkoutVersionBranch
+  versionIsIncremented
+  replaceFieldSed "${YAMLFILE}" 'prod-ver-major' "\"${MAJOR}\""
+  replaceFieldSed "${YAMLFILE}" 'prod-ver' "\"${MAJOR}.${MINOR}\""
+  replaceFieldSed "${YAMLFILE}" 'prod-ver-patch' "\"${MAJOR}.${MINOR}.${PATCH}\""
+  replaceFieldSed "${YAMLFILE}" 'prod-prev-ver' "\"${MAJOR}.$((MINOR - 1))\""
+  gitTag
+  gitPush ${MAJOR}.${MINOR}.x
+  echo "[INFO] Finished handling version update on branch: ${MAJOR}.${MINOR}.x"
+}
+
+Patch
 # Validate the version format.
 versionFormatIsValid
 
 # Get a working copy if necessary.
 gitClone
 
 # Update version in the version branch and in the main branch.
-for TARGET_BRANCH in "${MAJOR}.${MINOR}.x" "release-${VERSION}"
-do
-  gitBranch
-  versionUpdate
-  gitCommit
-  gitPush
-  case ${TARGET_BRANCH} in
-    "${MAJOR}.${MINOR}.x")
-      gitTag
-      ;;
-    "release-${VERSION}")
-      gitPullRequest
-      ;;
-  esac
-done
+if [[ ${PATCH} -eq 0 ]]; then
+  patchVersionUpdate
+  minorVersionUpdate
```

Comment:
Simply the release process by not doing changes in the main branch. See https://github.com/eclipse/che-docs/pull/2208/files#r783183143.

```suggestion
```